certified-mail-sender/src/main/resources/application.conf

```diff
@@ -0,0 +1,17 @@
+certified-mail-sender {
+  queue {
+    url = ${PEC_QUEUE_URL}
+    max-number-of-messages = 1
```

Comment:
what is this configuration used for? is it the max batch size?

Reply:
Yes, is the max batch size.
Here I would like to be sure that every mail sent will be removed from the queue in order to prevent multiple sending in case of errors.

Reply:
makes totally sense, may I ask to rename it to something like `max-consume-batch-size` instead?